vagrant/README.md

```diff
@@ -1,3 +1,10 @@
+
+## WARNING: kubernetes-anywhere is deprecated and will be retired in a future release.
+
+Kubeadm-playground was designed as a tool for helping kubeadm contributors.
+It will be discontinued soon in favor of https://github.com/kubernetes-sigs/kind 
```

Comment:
How about using this format for the link: `[kind](https://github.com/kubernetes-sigs/kind)`?

Reply:
Makes sense. Guess `kind` might want to change its name to `kINd` :smile_cat:

Reply:
Can we just change url to the annotated link,
i.e. _It will be discontinued soon in favor of_ [Kubernetes IN Docker](https://github.com/kubernetes-sigs/kind)

Reply:
This SGTM :+1: :100: